studio/components/interfaces/Billing/Subscription/Subscription.tsx

```diff
@@ -1,58 +1,56 @@
 import dayjs from 'dayjs'
-import { FC } from 'react'
+import { FC, useEffect } from 'react'
 import { useRouter } from 'next/router'
 import { Button, Loading } from 'ui'
 import * as Tooltip from '@radix-ui/react-tooltip'
 import { PermissionAction } from '@supabase/shared-types/out/constants'
 
-import { checkPermissions, useFlag } from 'hooks'
+import { checkPermissions, useFlag, useStore } from 'hooks'
 import { useParams } from 'common/hooks'
-import { STRIPE_PRODUCT_IDS } from 'lib/constants'
+import { PROJECT_STATUS, STRIPE_PRODUCT_IDS } from 'lib/constants'
 import { formatBytes } from 'lib/helpers'
 
 import { PRICING_TIER_PRODUCT_IDS } from 'lib/constants'
 import CostBreakdownRow from './CostBreakdownRow'
-import { StripeSubscription } from './Subscription.types'
 import NoPermission from 'components/ui/NoPermission'
 import { USAGE_BASED_PRODUCTS } from 'components/interfaces/Billing/Billing.constants'
 import { ProjectUsageResponseUsageKeys, useProjectUsageQuery } from 'data/usage/project-usage-query'
+import { useProjectSubscriptionQuery } from 'data/subscriptions/project-subscription-query'
 
 interface Props {
-  project: any
-  subscription: StripeSubscription
-  loading?: boolean
   showProjectName?: boolean
-  currentPeriodStart: number
-  currentPeriodEnd: number
 }
 
-const Subscription: FC<Props> = ({
-  project,
-  subscription,
-  loading = false,
-  showProjectName = false,
-  currentPeriodStart,
-  currentPeriodEnd,
-}) => {
+const Subscription: FC<Props> = ({ showProjectName = false }) => {
   const router = useRouter()
-
+  const { ui } = useStore()
   const { ref: projectRef } = useParams()
   const projectUpdateDisabled = useFlag('disableProjectCreationAndUpdate')
+  const isActive = ui.selectedProject?.status === PROJECT_STATUS.ACTIVE_HEALTHY
```

Comment:
Could also consider making a new hook in `ProjectContext.tsx` called, `useIsProjectActive()`:
```ts
const useIsProjectActive = () => {
  const { project } = useProjectContext()
  return project?.status === PROJECT_STATUS.ACTIVE_HEALTHY
}
```

Useful enough to be it's own hook? wdyt?